jenkinsci/blueocean-plugin
# Feature/jenkins 36582 favorites realtime

# Description
- See [JENKINS-36582](https://issues.jenkins-ci.org/browse/JENKINS-36582).
- Testing steps:
 - Add several favorites to the dashboard, ideally a simple Pipeline job, multi-branch Pipeline job, and a Freestyle job.
 - Using the classic UI, launch the jobs in those order. Note that the first two jobs should begin running while the Freestyle job is queued.
 - Jobs should update with the correct status as the builds complete.
 - Refreshing the page in the middle of runs should render jobs in correct status.
 - Navigating away and back should render jobs in correct status.

# Submitter checklist

# Reviewer checklist

@jenkinsci/code-reviewers @reviewbybees
